docs/Using-Fleet/Permissions.md

```diff
@@ -72,28 +70,28 @@ Users that are members of multiple teams can be assigned different roles for eac
 
 The following table depicts various permissions levels in a team.
```

Comment:
```suggestion
The following table depicts permissions levels in a team.

All hosts, software, policies, etc. are exclusive to one team.
```
TODO workshop this. We're trying to communicate that everything is this table is only accessible if the user is a member of that team.